src/GrammarApplicator_runRules.cpp

```diff
@@ -288,10 +291,10 @@ Reading* GrammarApplicator::get_sub_reading(Reading* tr, int sub_reading) {
 				mappings->push_back(tter);                                           \
 			}                                                                        \
 			else {                                                                   \
-				hash = addTagToReading((reading), tter);                             \
+				hash = addTagToReading(reading, tter);          \
```

Comment:
Those `()` were intentional. Macro arguments are text replacement, so it is recommended to always have them in extra `()`